packages/admin-theme/src/getTheme.ts

```diff
@@ -0,0 +1,52 @@
+import "@comet/admin/src/themeAugmentation";
+import "@comet/admin-rte/src/themeAugmentation";
+import "@comet/admin-react-select/src/themeAugmentation";
+import "@comet/admin-color-picker/src/themeAugmentation";
+
+import { createMuiTheme } from "@comet/admin";
+import { Theme } from "@material-ui/core/styles";
+
+import cometAdminColorPickerOverrides from "./cometAdminColorPickerOverrides/colorPicker";
+import cometAdminClearInputButtonOverrides from "./cometAdminOverrides/clearInputButton";
+import cometAdminFormFieldContainerOverrides from "./cometAdminOverrides/formFieldContainer";
+import cometAdminInputBaseOverrides from "./cometAdminOverrides/inputBase";
+import cometAdminMasterLayoutOverrides from "./cometAdminOverrides/masterLayout";
+import cometAdminMenuOverrides from "./cometAdminOverrides/menu";
+import cometAdminMenuCollapsibleItemOverrides from "./cometAdminOverrides/menuCollapsibleItem";
+import cometAdminMenuItemOverrides from "./cometAdminOverrides/menuItem";
+import cometAdminRteBlockTypeControlsOverrides from "./cometAdminRteOverrides/blockTypeControls";
+import cometAdminRteControlButtonOverrides from "./cometAdminRteOverrides/controlButton";
+import cometAdminRteFeaturesButtonGroupOverrides from "./cometAdminRteOverrides/featuresButtonGroup";
+import cometAdminRteLinkControlsOverrides from "./cometAdminRteOverrides/linkControls";
+import cometAdminRteRteOverrides from "./cometAdminRteOverrides/rte";
+import cometAdminRteToolbarOverrides from "./cometAdminRteOverrides/toolbar";
+import cometAdminSelectOverrides from "./cometAdminSelectOverrides/select";
+
+export const getTheme = (): Theme => {
+    return createMuiTheme({
```

Comment:
shouldon't we expose a createMuiTheme to allow easier overrides in application that wants to use the comet theme but customize it?

Reply:
@nsams yes, i think it would be best to just export the theme-options-object, instead of the created theme.  
Then you could merge the comet-theme-options with your own theme-options to create the theme. 

e.g.:
```tsx
import merge from "lodash/merge";
import { createMuiTheme, Theme, ThemeOptions } from "@material-ui/core";
import { getThemeOptions } from "@comet/admin-theme";

const cometThemeOptions: ThemeOptions = getThemeOptions();

const customThemeOptions: ThemeOptions = {
    // ...
}

const customizedCometTheme: Theme = createMuiTheme(merge(cometThemeOptions, customThemeOptions));
```

Reply:
yes, good idea.

I think even `createMuiTheme(cometThemeOptions, customThemeOptions);` would work (according to mui docs)